Userland/Libraries/LibWeb/DOM/Element.cpp

```diff
@@ -70,6 +70,8 @@
 #include <LibWeb/WebIDL/DOMException.h>
 #include <LibWeb/WebIDL/ExceptionOr.h>
 
+#pragma clang optimize off
```

Comment:
I'm assuming this wasn't supposed to make it into the commit 😄 (but if I'm wrong, a comment might be nice)

Reply:
Oopsie! :sweat_smile: